scripts/index.js

```diff
@@ -164,4 +164,29 @@ function callNTimes(time, fun) {
     }
 }
 
-callNTimes(5, helloWorld);
\ No newline at end of file
+callNTimes(5, helloWorld);
+
+// Str Multiply
+// Write a strMultiply function which takes two arguments:
+// str - the string to multiply
+// times - number of times to multiply
+// You may not use the native repeat method that strings provide. But you can use the range function defined below.
+
+function strMultiply(str, times) {
+    let sentence = '';
+    range(0, times).forEach(function () {
```

Comment:
Nice! I like that you chained on the `.forEach` directly to the result of calling `range()`.